src/StoreApi/Routes/CartUpdateCustomer.php

```diff
@@ -128,13 +128,13 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 	}
 
 	/**
-	 * If there is a draft order, update customer data there also.
+	 * If there is a draft order, update the customer data within that also so the
+	 * cart and order are kept in sync.
 	 *
 	 * @return void
 	 */
 	protected function maybe_update_order() {
```

Comment:
[Initially](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5323#discussion_r763199443) you renamed this to maybe_update_order_from_customer, is this something that you still want? Or the comment update above is enough?

Reply:
I've moved this to the PR where I implemented the new action hooks.